certbot/cli.py

```diff
@@ -991,10 +991,18 @@ def prepare_and_parse_args(plugins, args, detect_defaults=False):  # pylint: dis
         default=flag_default("tls_sni_01_port"),
         help=config_help("tls_sni_01_port"))
     helpful.add(
+        ["testing", "standalone", "apache", "nginx"], "--tls-sni-01-addr",
+        default=flag_default("tls_sni_01_addr"),
+        help=config_help("tls_sni_01_addr"))
+    helpful.add(
         ["testing", "standalone", "manual"], "--http-01-port", type=int,
         dest="http01_port",
         default=flag_default("http01_port"), help=config_help("http01_port"))
     helpful.add(
+        ["testing", "standalone", "manual"], "--http-01-addr",
```

Comment:
Honestly, I don't know either. I copied and pasted the option declaration for --http-01-port without understanding that part of it. @pde touched that line last, perhaps he can tell us how this should look?